src/lib/registration/clients/online_search.rb

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) [2019] SUSE LLC
```

Comment:
It's already 2020... :wink: 
```suggestion
# Copyright (c) [2020] SUSE LLC
```

Reply:
Hehe, you are right, time flies :smile:. I started to implement this feature in December 2019, hence the reason to have outdated headers... I will fix them.